cmake/StaticAnalyzers.cmake

```diff
@@ -0,0 +1,38 @@
+function(enable_static_analyzers target_name)
```

Comment:
The name of that function doesn’t exactly do what it’s implemented with — i.e. it can do nothing.

It can be good to isolate the scope of the target setting, and move the enable/disable logic to somewhere else.

Reply:
I agree the function name isn't that much relevant; do you have a suggestion ?

Since `StaticAnalyzers.cmake` is a helper, it have to hold the options and the logic itself.